support-files/mysql.spec.sh

```diff
@@ -246,7 +180,7 @@
 %define license_files_server    %{src_dir}/LICENSE.mysql
 %define license_type            Commercial
 %else
-%define license_files_server    %{src_dir}/COPYING %{src_dir}/README
+%define license_files_server    COPYING README
```

Comment:
This had to be changed due to change in %setup directive, those files were not found from the specified locations at some point. Need to check if it was a temporary issue.

Anyway, is there a difference between files found from under %{src_dir} and under build dir?